build-system/tasks/update-packages.js

```diff
@@ -175,7 +175,12 @@ function runYarnCheck() {
     const verifyTreeCmd = yarnExecutable + ' check --verify-tree';
     exec(verifyTreeCmd);
     log('Running', colors.cyan('yarn'), 'to update packages...');
-    execOrDie(yarnExecutable); // Stop execution when Ctrl + C is detected.
+
+    const installCmd = yarnExecutable;
```

Comment:
Make this a `let` instead of `const` because you're modifying the value two lines after